modular_skyrat/master_files/code/modules/mob/living/simple_animal/friendly/dogs.dm

```diff
@@ -155,20 +170,33 @@
 	. = ..(gibbed)
 	if(!.)
 		return FALSE
+
+	UnregisterSignal(src, COMSIG_ATOM_ATTACK_HAND)
+	UnregisterSignal(src, COMSIG_PARENT_ATTACKBY)
+	UnregisterSignal(src, COMSIG_ATOM_HITBY)
+	UnregisterSignal(src, COMSIG_ATOM_EMAG_ACT)
+
 	do_sparks(3, 1, src)
 	var/datum/ai_controller/dog/EN = ai_controller
 	LAZYCLEARLIST(EN.current_behaviors)
 
-/mob/living/simple_animal/pet/dog/corgi/borgi/emag_act(user as mob)
+/mob/living/simple_animal/pet/dog/corgi/borgi/proc/on_emag_act(mob/user)
 	if(!emagged)
 		emagged = 1
-		visible_message(span_warning("[user] swipes a card through [src]."), span_notice("You overload [src]s internal reactor."))
-		addtimer(CALLBACK(src, .proc/explode), 60 SECONDS)
+
+		emote("exclaim")
```

Comment:
Is that the right emote to do there? I don't even know if that's an emote we normally have, it's usually a speech modification.

Reply:
i think that exclaim is the one from ooz's emote pr